src/main/webapp/scripts/view.js

```diff
@@ -0,0 +1,36 @@
+export { authorizeButton, signoutButton, commentButton, videoId, commentElement, resultContainer };
+export { successfulApiCallback, failedApiCallback, updateSigninStatus }
+
+function updateDom(text, containerDOM) {
+    containerDOM.innerHTML = '';
```

Comment:
are you using this line to empty the container? It may be unsafe to use innerHTML, instead you can use a loop that's like while there's children of the element, remove that child. I use this method in other code so it may be useful to have that as a general helper function in script.js

Reply:
I don't see why it is unsafe since no user input is involved here with innerHtml, but I did find that while loop is faster at clearing child nodes. I will update that